test/stats.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-env mocha */
+'use strict'
+
+const chai = require('chai')
+chai.use(require('dirty-chai'))
+const expect = chai.expect
+
+const createNode = require('./utils/node').createNode
+
+describe('libp2p', (done) => {
+  it('has stats', () => {
+    createNode('/ip4/0.0.0.0/tcp/0', {
```

Comment:
Probably want to set address to `127.0.0.1` instead, no reason to bind to public interface :)

Reply:
@VictorBjelkholm good call, thanks! Fixed by 10bd08bdd51f1beaf0856d366e6f3f79bbf1a09f.